src/main/java/uk/gov/crowncommercial/dts/scale/service/agreements/model/entity/LotRelatedLot.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.crowncommercial.dts.scale.service.agreements.model.entity;
+
+import java.io.Serializable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import org.hibernate.annotations.Immutable;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.experimental.FieldDefaults;
+
+@Entity
+@Immutable
+@Table(name = "lot_related_lots")
```

Comment:
Not quite sure about the table naming but not an issue for here..

Reply:
Yep - found that quite confusing at the time as it's linking via Lot Rule